### PR TITLE
Feature create user

### LINKED ISSUE #19: Create user
User story #3

### PR DESCRIPTION
**Pull request by Sagi-Viktor**
Closes #19

### CODE REVIEW

File: src/main/java/com/codecool/terraformingmarsforum/controller/UserController.java [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+
 @RestController
 @RequestMapping("/api/users")
+@RequiredArgsConstructor
 public class UserController {
 
+    private final UserService userService;
+
+    @PostMapping("/")
+    public ResponseEntity<?> createUser(@RequestBody User user) {
```
Comment (benedekhalaj) [inline src/main/java/com/codecool/terraformingmarsforum/controller/UserController.java]: In this case, you send back a 'User' object, so return type should be 'ResponseEntity<User>

File: src/main/java/com/codecool/terraformingmarsforum/service/UserService.java [hunk 2]
```diff
@@ -12,4 +13,8 @@ public class UserService {
 
     @NonNull
     private UserDAO userDAO;
```
Comment (benedekhalaj) [inline src/main/java/com/codecool/terraformingmarsforum/service/UserService.java:15]: I think this is still field injection, but is it your part anyways?

**Review by benedekhalaj — CHANGES_REQUESTED** [review]
I request changes, but it is nice to see a test :) Great job

File: src/main/java/com/codecool/terraformingmarsforum/service/UserService.java (hunk 2 above)
Comment (Sagi-Viktor) [reply src/main/java/com/codecool/terraformingmarsforum/service/UserService.java:15]: Ask Bende about that, I dont want to change it, bc thats not my concept.
But yeah, you maybe rigth

File: src/main/java/com/codecool/terraformingmarsforum/controller/UserController.java (hunk 1 above)
Comment (Sagi-Viktor) [reply src/main/java/com/codecool/terraformingmarsforum/controller/UserController.java]: Done

**Review by benedekhalaj — APPROVED** [review]
Good job, approved! :)

**Pull request merged**